sites/platform/src/add-services/network-storage.md

```diff
@@ -298,14 +200,9 @@ For that case, all that's needed is to set the necessary file mounts as `service
 For example, the following `{{< vendor/configfile "app" >}}` file (fragment) keeps Drupal files directories shared between web and worker instances while keeping the Drush backup directory web-only (as it has no need to be shared).
 (This assumes a Network Storage service named `files` has also been defined in `{{< vendor/configfile "services" >}}`.)
 
-{{< version/specific >}}
-<!-- Platform.sh -->
-
 ```yaml {configFile="app"}
-{{< snippet name="my-app" config="app" >}}
-
 # The type of the application to build.
-type: "php:{{% latest "php" %}}"
+type: "php:8.3"
```

Comment:
Why remove the latest sc?

Reply:
```suggestion
"php:{{% latest "php" %}}"
```

Reply:
The only explanation I have is I must have copy-pasted an older version of the content by mistake  🤷‍♀️ .